theme/themes/pastanaga/extras/grid.less

```diff
@@ -0,0 +1,405 @@
+// Grid > Grid.Columns support, so we have to build our own because of the
+// Blocks Editor wrappers
+.block.gridBlock .grid-items [data-rbd-droppable-id] {
+  display: flex;
+}
+
+// general spacing in the Row block wrapper
+.block.gridBlock [class^='block-editor-'] {
+  padding-right: 1rem;
+  padding-left: 1rem;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+}
+
+// general spacing in the Column block wrapper
+.block.column [class^='block-editor-'] {
+  padding-right: 1rem;
+  padding-left: 1rem;
+  margin-top: 1rem;
+}
+
+.block.column [class^='block-editor-'] {
+  &:last-child {
+    margin-bottom: 0;
+  }
+}
+
+.block.gridBlock .one.grid-items [class^='block-editor-'] {
+  width: 100%;
+}
+
+.block.gridBlock .two.grid-items [class^='block-editor-'] {
+  width: 50%;
+}
+
+.block.gridBlock .three.grid-items [class^='block-editor-'] {
+  width: 33.33333333%;
+}
+
+.block.gridBlock .four.grid-items [class^='block-editor-'] {
+  width: 25%;
+}
+
+// Outter cell item flexbox behavior so all have the same height
+.block.gridBlock [class^='block-editor-'] {
+  display: inline-flex !important;
+  flex-direction: column;
+  align-self: stretch;
+}
+
+// (Inner) cell wrapper flexbox behavior so all have the same height
+.cell-wrapper {
+  position: relative;
+  flex-grow: 1;
+}
+
+// Add our own block selected marker different than the normal one so we can full control over it
+.block.gridBlock,
+.block.column {
+  [class^='block-editor-'].selected .cell-wrapper::before {
+    position: absolute;
+    // otherwise, this layer is above the object, then it does not allow clicking inside
+    z-index: -1;
+    top: -5px;
+    left: -5px;
+    display: block;
+    width: calc(100% + 10px);
+    height: calc(100% + 10px);
+    border: 1px solid rgba(120, 192, 215, 0.5);
+    border-radius: 3px;
+    content: '';
+  }
+}
+
+// Remove default bottom image block margin
+.block.gridBlock [class^='block-editor-'] .block.image {
+  margin: 0;
+}
+
+// Remove the text block "add block" button TODO: remove when we move to Quanta Toolbar
+.block.gridBlock [class^='block-editor-'] .block-add-button {
+  display: none;
+}
+
+// Remove completely the default inner selection marker, so we are using our own
+.block.gridBlock,
+.block.column {
+  [class^='block-editor-'] .block.selected::before {
+    content: unset;
+  }
+}
+
+// Disable hover blue border on inner blocks
+.block.gridBlock [class^='block-editor-'],
+.block.column [class^='block-editor-'],
+[class$='Row'].block [class^='block-editor-'] {
+  .block:not(.inner):not(.selected)::before {
+    content: unset;
+  }
+}
+
+// Remove block selection blue outline on hover for view only
+.block.gridBlock,
+.block.column {
+  :not([class^='block-editor-']) .block:hover::before {
+    border: none;
+  }
+}
+
+.block.gridBlock,
+.block.column {
+  .block.inner.slate {
+    cursor: text;
+  }
+
+  body.cms-ui .ui.basic.segment.content-area & {
+    cursor: grab;
+  }
+}
+
+// Aspect ratio on all images
```

Comment:
This might be too opinionated.

Reply:
It's configurable, maybe we should include docs for it.

Reply:
Included.